src/openrct2/windows/top_toolbar.c

```diff
@@ -98,6 +98,8 @@ typedef enum {
 	DDIDX_LAND_HEIGHTS = 10,
 	DDIDX_TRACK_HEIGHTS = 11,
 	DDIDX_PATH_HEIGHTS = 12,
+	DDIDX_VIEW_CLIPPING = 14,
+	TOP_TOOLBAR_VIEW_MENU_COUNT
```

Comment:
Just to make it a bit more obvious that the count is a special part of the `enum`, can you add a blank line inbetween?